styles/atom-ide-outline.less

```
@@ -83,20 +84,65 @@
   .icon {
     contain: @contain_except_size;
     display: inline-block;
-    width: 40px;
+    width: 5ch;
```

This also fixes the issue for me
```suggestion
    width: 8ch;
```